README.md

```diff
@@ -190,15 +177,31 @@ pub fn shave_all(yaks: usize) -> usize {
 tracing = "0.1"
 ```
 
-Note: Libraries should *NOT* call `set_global_default()`, as this will cause
-conflicts when executables try to set the default later.
+Note: Libraries should *NOT* install a subscriber by using a method than calls
+`set_global_default()`, as this will cause conflicts when executables try to
+set the default later.
 
 ### In Asynchronous Code
 
-If you are instrumenting code that make use of
-[`std::future::Future`][std-future] or async/await, be sure to use the
-[`tracing-futures`] crate. This is needed because the following example _will
-not_ work:
+To trace asynchronous code, the preferred method is using the [`#[instrument]`] attribute:
```

Comment:
how about
```suggestion
To trace `async fn`s, the preferred method is using the [`#[instrument]`] attribute:
```
so that we don't give the incorrect impression that `#[instrument]` also works with `async` blocks?

Reply:
Done.